lib/bpk-theming/src/create-with-theme.js

```diff
@@ -30,6 +31,14 @@ const createWithTheme = () => {
 
   return Component => {
     class WithTheme extends React.Component {
+      static propTypes = {
+        forwardedRef: PropTypes.func,
+      };
+
+      static defaultProps = {
+        forwardedRef: null,
```

Comment:
Maybe we should make this `undefined` by default? E.g. if ref is not provided it won't be set. At least we avoid the snapshot pollution with undefined.

Reply:
Good idea, will change.

Reply:
Done - and reverted snapshot changes as a result 👍